airflow/config_templates/config.yml

```diff
@@ -1709,6 +1709,14 @@
       type: float
       example: ~
       default: "15.0"
+    - name: clean_tis_without_dag
```

Comment:
Should we add a `sec` or `interval` suffix like other configs?

Reply:
Yep!